shared/git/new-repo/container.js

```diff
@@ -24,7 +24,10 @@ const mapDispatchToProps = (dispatch: any, {navigateAppend, navigateUp, routePro
       : GitGen.createCreatePersonalRepo({name})
     dispatch(createAction)
   },
-  onNewTeam: () => dispatch(navigateTo([teamsTab], ['showNewTeamDialog'])),
+  onNewTeam: () => {
+    dispatch(navigateTo(['showNewTeamDialog'], [teamsTab]))
```

Comment:
Couldn't you just do `navigateTo([teamsTab, 'showNewTeamDialog'])`

Reply:
fixed, tnx!